source/_integrations/iotawatt.markdown

```diff
@@ -12,8 +12,41 @@ ha_codeowners:
 ha_platforms:
   - sensor
 ---
-A sensor platform for the [IoTaWatt](https://www.iotawatt.com/) Open WiFi Electricity Monitor. It 
+A sensor platform for the [IoTaWatt](https://www.iotawatt.com/) Open WiFi Electricity Monitor. It
 will collect data from the Current Transformer Clamps (Input CTs) and any Outputs that are defined on the IoTaWatt
 and create them as sensors in Home Assistant.
 
 {% include integrations/config_flow.md %}
+
+## Energy management
+
+IoTaWatt does not provide the exact data that is needed for energy management. We're working with the IotaWatt team on resolving this.
+
+Until then, you can use these instructions to create the correct entities that work with energy management:
+
+### Configure IoTaWatt
+
+You will need to configure two new IoTaWatt output sensors:
+
+| Name | Unit | Formula
+| - | - | -
+| MainsConsumption|Watts|`(Main_In_Red + Main_In_White + Main_In_Blue) max 0`
+| MainsExport|Watts|`((Main_In_Red + Main_In_White + Main_In_Blue) min 0) abs`
```

Comment:
This kind of table really sucks when rendered.

Can we use a `{% configuration_basic %}` block instead?

Reply:
I guess not..